plugins/modules/files/ini_file.py

```diff
@@ -317,16 +433,31 @@ def main():
     section = module.params['section']
     option = module.params['option']
     value = module.params['value']
+    values = module.params['values']
     state = module.params['state']
     backup = module.params['backup']
     no_extra_spaces = module.params['no_extra_spaces']
     allow_no_value = module.params['allow_no_value']
     create = module.params['create']
 
-    if state == 'present' and not allow_no_value and value is None:
-        module.fail_json("Parameter 'value' must not be empty if state=present and allow_no_value=False")
+    if value is not None and values is not None:
+        module.fail_json(msg="Only 'value' or 'values' can be given, not both.")
```

Comment:
please use the `mutually_exclusive` specification in module's argument_spec definition. See https://docs.ansible.com/ansible/latest/reference_appendices/module_utils.html